package.json

```diff
@@ -9,7 +9,7 @@
         "express": "^4.16.3",
         "jquery-min": "0.0.2",
         "log-timestamp": "^0.1.2",
-        "pty": "0.0.0",
+        "node-pty": "^0.7.6",
```

Comment:
I don't know which version of node you're running, but it appears that `pty.js` is not updated for more recent versions. I'm running v10.0.0. It appears that `node-pty` is a suitable update so I've made the change (this fixed problems).